CHANGELOG.md

```diff
@@ -3,8 +3,9 @@
 ## [Unreleased]
 <details>
   <summary>
-    Changes that have landed in master but are not yet released.
-    Click to see more.
+
+- Add a monitoring of containers state while waiting for the web UI to open during initialization.
```

Comment:
The previous contents did not use the changelog categories because it was the initial release, but I think we should start from now on.
```suggestion
### Added

- Add a monitoring of containers state while waiting for the web UI to open during initialization ([#147](https://github.com/src-d/sourced-ce/issues/147)).
```